### PR TITLE
Search for opera.exe in various locations

### PR DESCRIPTION
**Pull request by Granjow**

### CODE REVIEW

**Comment by Granjow** [comment]
Any reasons not to merge?

**Comment by Granjow** [comment]
Thanks bot, CLA should be there now.

**Comment by Granjow** [comment]
Try again

**Comment by Granjow** [comment]
Awww, fine! Let's merge!

File: index.js [hunk 1]
```diff
@@ -58,13 +59,86 @@ var OperaBrowser = function(baseBrowserDecorator) {
...
+  var defaultPaths = [
+    process.env['ProgramFiles'],
+    process.env['ProgramFiles(X86)']
+  ];
+
+  var executable = null;
+  var found = defaultPaths.some(function (progFiles){
+    var oP = path.join(progFiles, 'Opera');
+    try {
+      //console.log('Checking ' + oP + ' ...');
```
Comment (Dignifiedquire) [inline index.js]: Please remove these log statements

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "grunt-auto-release": "~0.0.3"
   },
   "contributors": [
-    "Friedel Ziegelmayer <friedel.ziegelmayer@gmail.com>"
+    "Friedel Ziegelmayer <friedel.ziegelmayer@gmail.com>",
+    "Simon A. Eugster <simon.eu@gmail.com>"
```
Comment (Dignifiedquire) [inline package.json]: Please remove all these changes, they are done automatically when a release happens.

**Comment by Dignifiedquire** [comment]
Sorry for the late feedback, I left some inline comments, also could you please update the commit message to follow our [convention](http://karma-runner.github.io/0.12/dev/git-commit-msg.html)

**Comment by Granjow** [comment]
Opened #11 with your feedback implemented, I hope I have caught all of it!

**Pull request closed without merging**